src/screens/Accounts/Accounts.js

```diff
@@ -389,13 +406,35 @@ class AccountsScreen extends React.Component<Props, State> {
       accounts,
     } = this.props;
 
-    const ppnNetwork = blockchainNetworks.find((network) => network.id === BLOCKCHAIN_NETWORK_TYPES.PILLAR_NETWORK);
-    const availableStakeFormattedAmount = formatMoney(availableStake);
+    const ppnNetwork = blockchainNetworks.find(
+      (network) => network.id === BLOCKCHAIN_NETWORK_TYPES.PILLAR_NETWORK,
+    );
     const hasAccount = userHasSmartWallet(accounts);
     const hasSmartWallet = hasAccount && smartWalletFeatureEnabled;
 
+    const bitcoinNetwork = blockchainNetworks.find(
+      (network) => network.isAvailable && network.id === BLOCKCHAIN_NETWORK_TYPES.BITCOIN,
+    );
+
+    if (bitcoinNetwork) {
+      networks.push({
```

Comment:
Bitcoin card should go after Pillar Network card :)

Reply:
Updated